src/argilla/server/database.py

```diff
@@ -12,12 +12,24 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-from sqlalchemy import create_engine
+from sqlite3 import Connection as SQLite3Connection
```

Comment:
Is `sqlite3` installed as default? or is an optional dep from sqlalchemy. If second, we should validate that the database module still works if sqlite is not installed (using another db engine)

Reply:
It's part of the python standard lib. Looks like SQLAlchemy is internally using it to construct the db connection.